examples/sync-merkleroots.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  ErrorNoAdminKey,
+  ErrorResponse,
+  MerkleRoot,
+  Repository,
+  SpvWalletClient,
+  SpvWalletError,
+} from '../dist/typescript-npm-package.cjs.js';
+import { exampleXPub } from './example-keys.js';
+import { errMessage } from './utils.js';
+
+const server = 'http://localhost:3003/api/v1';
+
+// perhaps change exmapleXPub to actual existing in db
+if (!exampleXPub) {
+  console.log(errMessage('xPub'));
+  process.exit(1);
+}
+
+const client = new SpvWalletClient(server, {
+  xPub: exampleXPub,
+});
+
+// simulate database
+const db: {
+  merkleRoots: MerkleRoot[];
+} = {
+  merkleRoots: [
+    {
+      merkleRoot: '4a5e1e4baab89f3a32518a88c31bc87f618f76673e2cc77ab2127b7afdeda33b',
+      blockHeight: 0,
+    },
+    {
+      merkleRoot: '0e3e2357e806b6cdb1f70b54c3a3a17b6714ee1f0e68bebb44a74b1efd512098',
+      blockHeight: 1,
+    },
+    {
+      merkleRoot: '9b0fc92260312ce44e74ef369f5c66bbb85848f2eddd5a7a1cde251e54ccfdd5',
+      blockHeight: 2,
+    },
+    {
+      merkleRoot: '612209eca3ff078e55d1ffe4602e78ac9a53458c7f9196b38c232d8af9ed635d',
+      blockHeight: 864550,
+    },
```

Comment:
I suppose you wanted to speed it up - that's why you've skipped majority of blocks and put right here `864550`.
But... for the sake of "documenting things" I'm not sure if somebody out of our team would understand why here are `0, 1, 2, 864550`. 
- I would put here only `0, 1, 2` and let the sync work "normally"
- Also, I would put some more descriptive comment here: `// simulate database`

Reply:
done